sdk/python/tests/test_historical_retrieval.py

```diff
@@ -274,6 +286,7 @@ def test_historical_features_from_parquet_sources():
                     "customer_id",
                 ]
             ).reset_index(drop=True),
+            check_dtype=False,
```

Comment:
Is there a reason for this change?

Reply:
There were errors for 32bit vs 64 bit stuff, and didn't think it was important to change. I can look into it if you're concerned though